### PR TITLE
fix network CLI argument being ignored

### PR DESCRIPTION
**Pull request by louisguitton**
I was facing this error and I couldn't figure out why, so I read the code and I think I found a 🐛 bug.
The network CLI param is not used it seems.
```sh
→ fresh deploy --network 'testnet'
⠼ Deploying project to testnet ...Error: Command failed: flow project deploy         --network=emulator         -f flow.json         --update         -o json
❌ Command Error: failed to deploy all contracts
    at ChildProcess.exithandler (child_process.js:383:12)
    at ChildProcess.emit (events.js:400:28)
    at maybeClose (internal/child_process.js:1058:16)
    at Socket.<anonymous> (internal/child_process.js:443:11)
    at Socket.emit (events.js:400:28)
    at Pipe.<anonymous> (net.js:686:12) {
  killed: false,
  code: 1,
  signal: null,
  cmd: 'flow project deploy         --network=emulator         -f flow.json         --update         -o json',
  stdout: '\n',
  stderr: '❌ Command Error: failed to deploy all contracts'
}
```
specifically see the highlighting: the commander-js CLI argument is fine (the log says testnet) but the flow CLI command that is generated is using the wrong network.
image

Please guide me as to how I can test this PR 🙏

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -252,14 +252,14 @@ async function getNFT(tokenId) {
 }
 
 async function pinNFTData(tokenId) {
-  const fresh = await MakeFresh();
+  const fresh = await MakeFresh("emulator");
```
Comment (10thfloor) [inline src/index.js:255]: These shouldn't be hardcoded.

**Review by 10thfloor — APPROVED** [review]

**Review by 10thfloor — CHANGES_REQUESTED** [review]
Make sure all instance of MakeFresh use the configuration value for the network, instead of hardcoding.

**Comment by louisguitton** [comment]
> Make sure all instance of MakeFresh use the configuration value for the network, instead of hardcoding.

I'm not sure I got what you meant by "the configuration value" 100% (you meant from the CLI right? not `getConfig`?) . 
I made a change in the direction of removing hard coded values, and making network choice explicit as a CLI argument.

**Comment by louisguitton** [comment]
To test this change I did the following steps:
```sh
npm install
node src/index.js start
# name it example
cd example
# go to https://testnet-faucet.onflow.org/, create a test account
# change .env with the right values
node ../src/index.js deploy -n testnet
# open https://flow-view-source.com/testnet/account/0xyyyyyy
```
and it worked https://flow-view-source.com/testnet/account/0x0ef0811b45ba3f6e
image

**Comment by louisguitton** [comment]
I'm going from error to error unfortunately. Now it's when claiming the NFTs. It seems #9 broke the project.
```
[Error Code: 1101] cadence runtime error Execution failed:
error: unexpectedly found nil while forcing an Optional value
  --> 2aa305909d668f00e4ee7cd755a9d72000c773219352019e5987ee220294a736:31:49
   |
31 |             .borrow<&{NFTQueueDrop.DropPublic}>()!
   |                                                  ^
```

At this stage, I will just fork the project and revert changes up to the last working release and will add my network fix there.

**Comment by psiemens** [comment]
Ah, sorry about that @louisguitton. I'll see if I can fix those errors.

**Comment by louisguitton** [comment]
not to worry @psiemens and thanks so much for your fix 🙏 . 

I'll have to figure out in which order I rebase things because I'm mixing concerns in this PR after all (between the testnet contrib and the misc tiny fixes)...
Also, maybe the changes I've made in https://github.com/louisguitton/freshmint/pull/1/files (on top of the last commit before #9) also could be useful (e.g. the flow.json env variable parsing). I'll try to find time later this week to iron this out.

**Comment by psiemens** [comment]
Hey @louisguitton, @10thfloor and I have actually been wanting to get testnet working for a little while. Do you mind if we work on top of your fork today? This is really great stuff.

**Comment by louisguitton** [comment]
> Hey @louisguitton, @10thfloor and I have actually been wanting to get testnet working for a little while. Do you mind if we work on top of your fork today? This is really great stuff.

of course @psiemens please go ahead. Glad you find this helpful. I certainly find the `freshmint` project super helpful.
My suggestion would be too cherry pick the commits of my fork on top of `v1.0.1-alpha1` and release it as a hotfix, git tagged with `v1.0.2`. Then you could rebase #9 on top of that, resolve conflicts etc... and work toward v2.0

**Pull request closed without merging**